Makefile

```diff
@@ -17,7 +17,7 @@ BASE_IMAGE = gcr.io/distroless/static:nonroot
 ################################################################################
 
 GO ?= $(shell command -v go 2> /dev/null)
-PACKAGES=$(shell go list ./...)
+PACKAGES=$(shell go list ./... | grep -v vendor)
```

Comment:
Why is this change introduced?

Reply:
it's redundant to include vendor as it's 3rd party dependency with `go mod vendor` and breaks the lint if I aremember correctly